lib/apollo_tracing/pipeline.ex

```diff
@@ -5,17 +5,19 @@ defmodule ApolloTracing.Pipeline do
     |> add_phases()
   end
 
-  def plug(config, pipeline_opts \\ []) do
-    case Code.ensure_loaded(Absinthe.Plug) do
-      {:module, absinthe_plug} ->
-        absinthe_plug.default_pipeline(config, pipeline_opts)
-       |> add_phases()
-      _ ->
-        raise RuntimeError, """
-          You don't have Plug loaded, please use
-          ApolloTracing.Pipeline.default(absinthe_schema, pipeline_opts)
-          to produce a pipeline without Plug specific phases
-        """
+  if Code.ensure_loaded?(Absinthe.Plug) do
```

Comment:
This is unrelated to the other changes. `Code.ensure_loaded` can be a VERY expensive call, and should basically never happen at runtime. Always do the check at compile time and have it generate different functions.